packages/legacy/core/App/components/misc/UnorderedListModal.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
What's the difference between `packages/legacy/core/App/components/misc/UnorderedListModal.tsx` and `packages/legacy/core/App/components/misc/UnorderedList.tsx`? they appear to be the same component to me

Reply:
It's the text color. If the moda bgcolor is different from the app's bgcolor, the text might be difficult to see in some cases. So, I put a different text color in the theme for modal screens. Since bifold has the same bgcolor for modals and the app, the text color will be the same. But in our app, it's different.